portal-impl/src/com/liferay/portlet/documentlibrary/service.xml

```diff
@@ -95,6 +96,14 @@
 			<finder-column name="groupId" />
 			<finder-column arrayable-operator="OR" name="folderId" />
 		</finder>
+		<finder name="G_LtD" return-type="Collection">
+			<finder-column name="groupId" />
+			<finder-column comparator="&lt;" name="displayDate" />
+		</finder>
+		<finder name="C_LtD" return-type="Collection">
+			<finder-column name="companyId" />
+			<finder-column comparator="&lt;" name="displayDate" />
+		</finder>
```

Comment:
Just a question: why are you creating these finders now? Are you 100% we'll be using these in followup PRs?